apps/src/templates/Certificate.jsx

```diff
@@ -145,7 +146,7 @@ class Certificate extends Component {
           )}
           {tutorial && this.state.personalized && (
             <div>
-              <h2>{i18n.congratsCertificateThanks()}</h2>
+              <h2 id="thanks">{i18n.congratsCertificateThanks()}</h2>
```

Comment:
Elsewhere we've used [`uitest-`](https://github.com/code-dot-org/code-dot-org/search?q=uitest) as a prefix for IDs to make it clear that they're only used for testing, not for CSS or JS lookup.